cmd/minishift/cmd/start.go

```diff
@@ -87,6 +82,13 @@ var minishiftToClusterUp = map[string]string{
 
 // init configures the command line options of this command
 func init() {
+	startCmd = &cobra.Command{
```

Comment:
Why breaking consistency with previous implementations? Non of the commands have been defined like this inside the `init` function...